### PR TITLE
install ceph-common in control plane so rbd provisioner can find rbd cli to create rbd images

### PR DESCRIPTION
**Pull request by rootfs**
fix https://bugzilla.redhat.com/show_bug.cgi?id=1582059

### CODE REVIEW

File: images/node/Dockerfile [hunk 1]
```diff
@@ -16,14 +16,12 @@ COPY system-container/manifest.json system-container/config.json.template system
 RUN INSTALL_PKGS=" \
       origin-hyperkube origin-node origin-sdn-ovs \
       socat ethtool device-mapper iptables nmap-ncat e2fsprogs \
-      xfsprogs device-mapper-persistent-data ceph-common \
+      xfsprogs device-mapper-persistent-data\
```
Comment (smarterclayton) [inline images/node/Dockerfile]: Why did you remove this here?

Comment (sdodson) [reply images/node/Dockerfile]: Because node is FROM openshift/origin-control-plane

File: images/origin/Dockerfile [hunk 2]
```diff
@@ -6,7 +6,9 @@
 #
 FROM openshift/origin-cli
 
-RUN INSTALL_PKGS="origin" && \
+RUN INSTALL_PKGS="origin ceph-common" && \
```
[the remark this answers is not in the document]
Comment (jsafrane) [reply images/origin/Dockerfile]: It's used by controller-manager for dynamic provisioning and resize and by kubelet to map/unmap (local "attach"/"detach") volumes.

[the remark this answers is not in the document]
Comment (rootfs) [reply images/origin/Dockerfile]: yes we need ceph-common to pick up rbd command for controller to provision and resize rbd images.

**Comment by rootfs** [comment]
@smarterclayton @childsb yes we need ceph-common to pick up rbd command for controller to provision and resize rbd images.

File: images/node/Dockerfile (hunk 1 above)
Comment (smarterclayton) [reply images/node/Dockerfile]: It has to remain here.  Node image won’t depend on control plane next release

**Comment by rootfs** [comment]
@sdodson @smarterclayton thanks, review addressed. 

cc @bertinatto

File: images/node/Dockerfile [hunk 3]
```diff
@@ -15,15 +15,13 @@ COPY system-container/manifest.json system-container/config.json.template system
...
-      xfsprogs device-mapper-persistent-data ceph-common \
+      socat ethtool device-mapper iptables nmap-ncat e2fsprogs ceph-common \
+      xfsprogs device-mapper-persistent-data\
       libmnl libnetfilter_conntrack conntrack-tools \
       libnfnetlink iproute bridge-utils procps-ng openssl \
       binutils xz kmod-libs kmod sysvinit-tools device-mapper-libs dbus \
       iscsi-initiator-utils \
       " && \
-    yum install -y centos-release-ceph-luminous && \
-    rpm -V centos-release-ceph-luminous && \
```
Comment (smarterclayton) [inline images/node/Dockerfile]: You need to still install this here if it isn’t already.  You should remove all changes to this file.

**Comment by jsafrane** [comment]
@rootfs, can you please squash the commits?

**Comment by rootfs** [comment]
@jsafrane squashed
